src/containers/checkout/checkout-form/delivery/worldwide/worldwide.js

```diff
@@ -156,6 +158,7 @@ const Worldwide = ({ errors, touched, values, handleChange, setFieldValue }) =>
         <Autocomplete
           className={styles.addressInput}
           options={citiesOptions}
+          getOptionSelected={(option, value) => option === value || value === ''}
```

Comment:
Assign this function to a constant if you need to use it more than once.

Reply:
fixed